src/OAuth1/AbstractProvider.php

```diff
@@ -135,12 +135,17 @@ public function parseToken($body)
      * @param array $parameters
      * @return \SocialConnect\Common\Http\Response
      */
-    protected function oauthRequest($uri, $method = Client::GET, $parameters = [])
+    public function oauthRequest($uri, $method = Client::GET, $parameters = [])
     {
         $headers = [
             'Accept' => 'application/json'
         ];
 
+        if (isset($parameters['headers']) && is_Array($parameters['headers'])) {
```

Comment:
@heiglandreas What do you think, if We will create a new parameter/argument for headers?

![image](https://cloud.githubusercontent.com/assets/572096/24513991/b8d4b0ca-15ad-11e7-82c7-cd1d006b00ef.png)

Thanks